src/styles/utils/general.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  height: 100vh;
+}
```

Comment:
set all these properties for container, as I mentioned before. BEM doesn't allow to use tags name